geospaas_rest_api/tests.py

```diff
@@ -1,3 +1,146 @@
 from django.test import TestCase
+from django.test import Client
+from geospaas.vocabularies.models import Platform, Instrument 
+from geospaas.vocabularies.models import Parameter, DataCenter
+import json
 
 # Create your tests here.
+class DatasetURITests2(TestCase):
```

Comment:
Use duck-typing principle when giving names to classes and methods:
"it walks like a duck, it quack like a duck, it looks like a duck - it is a duck!"
I.e. give self explanatory names to classes, methods, variables.
For example:
`DatasetURITests2` is a wrong name. Because it does not test DatasetURI!
`test_api_initial_calls2` is a wrong name. Because it does not tell anything.